synapseclient/client.py

```diff
@@ -5247,3 +5270,146 @@ def _return_rest_body(self, response):
         if is_json(response.headers.get("content-type", None)):
             return response.json()
         return response.text
+
+
+@dataclass
+class Permissions:
+    """
+    The permission a User has for a given Entity.
+
+    Attributes:
+        can_view (Boolean): Can the user view this entity?
```

Comment:
Let's plan to not include types in the docstring as `mkdocstrings` will automatically grab this information based off the typehint of the field.